exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/transactions/ErrorTxIntegrationTest.java

```diff
@@ -102,6 +103,7 @@ void constructorRejectsInvalidDescription() {
         () -> new ErrorTx(1L, (byte) 1, invalidDescription));
   }
 
+  @Disabled
```

Comment:
I have already requested to clarify the reason tests are disabled.

Reply:
Hm, somehow I missed this group of tests, sorry...
These tests fail on violation of native Fork limitations with `multiple mutable borrows of an index at IndexAddress...` message.

Reply:
:warning: But there are also other such annotation in tests of services and testkit.

If it is believed that some tests fail for several reasons (e.g., broken root hash and multiple indexes), we might add a task to fix all the disabled tests _after all_ issues are closed :man_shrugging: But if (some of) the detective work has already been carried out, I would rather not repeat it.